hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -94,6 +94,7 @@ function ozonecmd_case
     classpath)
       if [[ "$#" -gt 0 ]]; then
         OZONE_RUN_ARTIFACT_NAME="$1"
+        OZONE_VALIDATE_CLASSPATH="true"
```

Comment:
These should be set, by `ozone_parse_opts`, only if `--validate` is present.

Reply:
Set `OZONE_VALIDATE_CLASSPATH` (=`true`) under `ozone_parse_opts` only if the `--validate` flag is passed, thanks